package-lock.json

```diff
@@ -0,0 +1,13900 @@
+{
```

Comment:
revert to your old package-lock.json or pull from main/develop, re adding the entire file is pointless